### DISCUSSION TITLE
Is there a good reason as to why the configuration files are exported to a .iconify file and not a standard file format like .ini or .XML?

### DISCUSSION THREAD

**Discussion by IdkWhatToCallMe123** [Mahmud0808/Iconify#393] [General]
title

**Comment by Mahmud0808** [comment]
Extension doesn't matter. You can rename it to literally anything and it will still work.

**Comment by IdkWhatToCallMe123** [comment]
My problem is that vs code seem to be hella confused on how to read the file 
![圖片](https://github.com/Mahmud0808/Iconify/assets/74187631/39e9b384-5d53-441d-9bc4-bbeb8b88807b)
It would be cool if the files would save in like a json, ini or xml format so could easily edit them on my computer where it is a bit easier to copy color values over from color palettes I find online.

**Comment by Mahmud0808** [comment]
Unfortunately you can't do that. It's java object output stream which is in binary format to prevent people from modifying the file. Otherwise ayone could modify and enable conflicting overlays which may result in SystemUI crash for someone else.
